cassandra/4.0/modules/ROOT/pages/cassandra-connector-examples.adoc

```diff
@@ -0,0 +1,575 @@
+    <http:listener-connection host="0.0.0.0" port="8081" />
+    </http:listener-config>
+  <cassandra-db:config name="CassandraDB_Config" doc:name="CassandraDB Config">
+    <cassandra-db:connection
+      host="${config.host}"
+      port="${config.port}"
+      keyspace="${config.keyspace}"
+      username="${config.username}"
+      password="${config.password}"/>
+	</cassandra-db:config>
+  <flow name="HTMLForm">
+    <http:listener doc:name="Listener" config-ref="HTTP_Listener_config" path="/"/>
+    <parse-template doc:name="Parse Template" location="form.html"/>
+  </flow>
+  <flow name="CreateKeyspace">
+    <http:listener
+      doc:name="Listener"
+      config-ref="HTTP_Listener_config"
+      path="/createKeyspace"/>
+    <ee:transform doc:name="Transform Message">
+      <ee:message >
+        <ee:set-payload ><![CDATA[%dw 2.0
+output application/java
+---
+{
+  "keyspaceName": payload.keyspaceName,
+  "replicationFactor": payload.replicationFactor,
+  "replicationStrategyClass": payload.replicationStrategyClass
+} as Object {
+  class : "org.mule.modules.cassandradb.api.CreateKeyspaceInput"
+}]]></ee:set-payload>
+      </ee:message>
+    </ee:transform>
+    <cassandra-db:create-keyspace doc:name="Create keyspace" config-ref="CassandraDB_Config"/>
+    <ee:transform doc:name="Transform Message">
+      <ee:message >
+        <ee:set-payload ><![CDATA[%dw 2.0
+output application/json
+---
+payload]]></ee:set-payload>
+      </ee:message>
+    </ee:transform>
+  </flow>
+  <flow name="CreateTable">
+    <http:listener doc:name="Listener" config-ref="HTTP_Listener_config" path="/createTable"/>
+    <ee:transform doc:name="Transform Message">
+      <ee:message>
+        <ee:set-payload><![CDATA[%dw 2.0
+output application/java
+---
+{
+  "columns": payload.columns,
+  "tableName": payload.tableName,
+  "keyspaceName": payload.keyspaceName
+} as Object {
+  class : "org.mule.modules.cassandradb.api.CreateTableInput"
+}]]></ee:set-payload>
+      </ee:message>
+    </ee:transform>
+    <cassandra-db:create-table doc:name="Create table" config-ref="CassandraDB_Config"/>
+    <ee:transform doc:name="Transform Message">
+      <ee:message>
+        <ee:set-payload><![CDATA[%dw 2.0
+output application/json
+---
+payload]]></ee:set-payload>
+      </ee:message>
+    </ee:transform>
+  </flow>
+</mule>
+----
+
+[[manipulate-data]]
+== Manipulate Data in an Apache Cassandra Database
+
+This example shows the following ways to manipulate data in an Apache Cassandra database:
+
+* Get table names from a keyspace.
+* Perform the following CRUD operations on data in an Apache Cassandra table:
+** Insert
+** Select
+** Update
+** Delete
+* Perform the following operations on an Apache Cassandra table column:
+** Delete the column's value.
+** Rename a column.
+** Add a new column.
+** Drop a column.
+* Execute a Cassandra Query Language (CQL) query.
+* Drop data from the Apache Cassandra database table.
+
```

Comment:
Here jumps directly to the XML sample of the application with the operations. I would add an intro line of this XML